### DISCUSSION TITLE
[10 gennaio 2024] Complessità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#21] [Complessità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/36816d84-557b-463e-a717-b36998a7a265)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/43db35f0-eeea-4a52-a4eb-52bdb838de4b)

-----

Il secondo punto, invece, è un diretto corollario del Teorema di Savitch, il quale può essere trovato sul libro di testo:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/89f1c832-fb5b-4938-861e-6ee69afc62e2)

**Comment by CasuFrost** [comment]
che esame regalato questo
